manifest-spec.md

```diff
@@ -1,37 +1,140 @@
   - abi
   - unlinked binary
-  - deployed address if applicable
-  - name/address pairs for linked libraries
+  - deployed addresses
+  - linked libraries
   - compiler version for bytecode verification
-  - key/value pairs of the sha3 hash of events this contract can emit with the abi definition of that event (i.e., the contract metadata should contain information for all events that can be triggered by this contract, including those triggered by linked libraries)
 - external project URIs (homepage, repository uri, etc.)
 - dependencies (i.e., references to other packages and versions)
-- updated time (likely added by the packager)
 
+## Manifest Data Specification
+
+### Manifest Version: `manifest_version`
+
+
+The `manifest_version` field defines the version of the ethereum package manifest
+specification (this document) that this manifest conforms to. All manifests
+**must** include this field.
+
+* Key: `manifest_version`
+* Type: Integer
+* Allowed Values: `1`
+* Package Manager Guide: Package managers should validate this field is valid prior to publishing new manifests.
+
+
+### Package Name: `package_name`
+
+The `package_name` field defines a human readable name for this package.  All
+manifests **should** include this field.
+
+* TODO: what should happen if this is missing (should it be required?)
+* Key: `package_name`
+* Type: String
+* Package Manager Guide: Package managers should use this field when registering new packages (TODO: define this better)
+
+### Author: `author`
+
+The `author` field defines a human readable name for the author of this package.  All
+manifests **should** include this field. 
 
-## Manifest format
 
-Package Manifests can be stored in the following formats:
+* Key: `author`
+* Type: String
 
-* JSON document in a `package.json` file.
+### Version: `version`
+
+The `version` field declares the version number of this release.  This value
+**should** be conform to the [semver](http://semver.org/) version numbering
+specification.  All manifests must **include** this field.
+
+* Key: `version`
+* Type: String
+* Package Manager Guide: Package Managers should validate this field conforms to the *semver* format and display a warning if it does not.
+
+
+### Description: `description`
+
+The `description` field *should* be used to provide additional detail that may be relevant for the package.
+
+* TODO: should this include a suggestion to use a specific markup format like markdown?
+* Key: `description`
+* Type: String
+
+
+### Links: `links`
+
+The `links` field *should* be used to provide URIs to relevant resources
+associated with this package.  When possible, authors *should* use the
+following keys for the following common resources.
+
+* `website`: Primary website for the package.
+* `documentation`: Package Documentation
+* `repository`: Location of the project source code.
+* TODO: what other common fields?
+
+* Key: `links`
+* Type: Hash(String: String)
+
+### Contract Meta: `??-TODO-??`
```

Comment:
Contract(s)
In case this is not a typo, why do you think they shouldn't be several contracts per package?